js/base.js

```diff
@@ -148,6 +155,19 @@ export class API {
         this.consumer = await adapters[this.adapter + "Consumer"].build(this, options);
         await this.consumer.connect();
     }
+
+    /**
+     * Parses a topic name to extract the global topic.
+     * For example, the global topic of "global:subdomain-123"
+     * is "global".
+     *
+     * @param {String} topic The name of the topic to be parse.
+     * @returns {String} The global topic name.
+     */
+    _getGlobalTopic(topic, separator = ":") {

```

Comment:
I had discussed with Joao once I think and he thought "get" implies some kind of networking or file IO.
Hence I would rename this to just "_globalTopic"